app/(main)/league/all/page.tsx

```diff
@@ -56,9 +58,7 @@ const Leagues = (): JSX.Element => {
           ))
         ) : (
           <div className="text-center">
-            <p className="text-lg font-bold">
```

Comment:
This is incorrect. The loader should replace to submit content as the API is waiting for a response. This message is correct when there are 0 leagues associated with the user.

Reply:
Removed from this page

Reply:
You reverted the <p> back, but the styles were not carried over.

Reply:
@shashilo can you elaborate? I think I've done what you're asking

Reply:
@shashilo what loader is meant to be used on the LeagueCard component? I thought this input spinner was for buttons only?

Reply:
@ryandotfurrer Was this question addressed?

Reply:
`<p className="text-lg font-bold">` These classes were removed. Wondering if this is intentional or not.

Reply:
> @ryandotfurrer Was this question addressed?

It was not. @shashilo, what loader is meant to be used on the LeagueCard component? I thought this input spinner was for buttons only?

Reply:
> `<p className="text-lg font-bold">` These classes were removed. Wondering if this is intentional or not.

These are back in and will be reflected upon the next commit.

Reply:
> > @ryandotfurrer Was this question addressed?
> 
> It was not. @shashilo, what loader is meant to be used on the LeagueCard component? I thought this input spinner was for buttons only?

@shashilo pinging you for a reminder, please advise.